webpack.config.js

```diff
@@ -36,6 +36,7 @@ const plugins = [
   new webpack.EnvironmentPlugin(['WORKSPACE_URL']),
   new webpack.EnvironmentPlugin(['WTS_URL']),
   new webpack.EnvironmentPlugin(['MANIFEST_SERVICE_URL']),
+  new webpack.EnvironmentPlugin(['MapboxAccessToken']),
```

Comment:
can we change the name of this env var to sth like `MAPBOX_ACCESS_TOKEN` so all env vars have similar format?

Reply:
https://github.com/uc-cdis/cloud-automation/pull/1200/files FYI